Tests/SwiftFusionTests/MCMC/MCMCTests.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  MCMCTests.swift
+//  SwiftFusionTests
+//
+//  Created by Frank Dellaert on 6/23/20.
+//
+
+import TensorFlow
+import SwiftFusion
+
+import XCTest
+
+class MCMCTests: XCTestCase {
+  
+  override func setUpWithError() throws {
+    // Put setup code here. This method is called before the invocation of each test method in the class.
+  }
+  
+  override func tearDownWithError() throws {
+    // Put teardown code here. This method is called after the invocation of each test method in the class.
+  }
+  
+  /// Sampling from the Standard Normal Distribution.
+  /// Inspired by testRWM1DUniform from tfp
+  func testRWM1DUniform() throws {
```

Comment:
Maybe add some explanation? There are no docs on the TFP side either, but we should document this.

Reply:
Good point. Done. Let me know if more clear now.